pkg/agave/assets/install.sh

```diff
@@ -76,6 +76,9 @@ step::80::setup-validator-startup() {
 
   cat <<EOF | $SUDO tee /home/sol/run-validator >/dev/null
 #!/usr/bin/env bash
+if [ -n "\$SOLANA_METRICS_CONFIG" ]; then
+  export SOLANA_METRICS_CONFIG="\$SOLANA_METRICS_CONFIG"
+fi
 exec agave-validator $VALIDATOR_FLAGS
```

Comment:
For this, to be consistent, have you thought about doing:

```
$VALIDATOR_ENV exec agave-validator $VALIDATOR_FLAGS
```

and then pass in VALIDATOR_ENV as a `strings.Join(anEnvMap, " ")` which would contain your `SOLANA_METRICS_CONFIG`.  It's less hoops to jump through if you ever want to add an additional environment variable.

Or you could create an `env` file like we do in other places, you could then source that.